app/models/donut.rb

```diff
@@ -1,11 +1,5 @@
 class Donut < ApplicationRecord
```

Comment:
ok there should be a `belongs_to` association here as well as a `Category` model

Reply:
addressed in latest commit